projects/plugins/social/src/js/components/admin-page/styles.module.scss

```diff
@@ -18,4 +18,19 @@ ul.jp-product-promote {
 		font-size: 16px;
 		line-height: 1.5;
 	} 
+}
+table {
```

Comment:
This will affect all tables on the admin page. Not an issue for now, but something to keep in mind.